WordPress/src/test/java/org/wordpress/android/ui/sitecreation/domains/SiteCreationDomainsViewModelTest.kt

```diff
@@ -130,6 +133,24 @@ class SiteCreationDomainsViewModelTest {
         }
     }
 
+    private fun <T> testWithInvalidQueryErrorResponse(
```

Comment:
I just noticed this can be removed by extending and reusing `testWithErrorResponse`.

Reply:
Updated ✅